packages/social-trader/src/pages/landing-page/components/accordion/accordion.tsx

```diff
@@ -19,11 +27,30 @@ interface Props {
 }
 
 const _Accordion: React.FC<Props> = ({ accordion, className }) => {
+  const { effectiveConnectionType = "4g" } = useNetworkStatus();
```

Comment:
No, in hook argument, like here:
```
const initialEffectiveConnectionType = '4g';
const { effectiveConnectionType } = useNetworkStatus(initialEffectiveConnectionType);
```
https://www.npmjs.com/package/react-adaptive-hooks
You can make const initialEffectiveConnectionType in _hooks/network-status_ and use it in all files